tests/grammar.rs

```diff
@@ -19,7 +19,7 @@ mod tests {
             <rule>           ::= <opt-whitespace> \"<\" <rule-name> \">\"
                                 <opt-whitespace> \"::=\" <opt-whitespace>
                                 <expression> <line-end>
-            <opt-whitespace> ::=  \"\" | \" \" <opt-whitespace>
+            <opt-whitespace> ::=  \"\" | \" \" 
```

Comment:
Does this change the meta BNF grammar to only generate grammars with 0 or 1 space between the `<rule-name>` and `::=` ? I thought it was legal to have more than 1.

Reply:
Yeah it does, I'm not sure it's something we should remove if we don't have to, it was just the likely candidate for a lot of recursion and I wanted to see if things all built without it